payment-request/payment-response/helpers.js

```diff
@@ -113,6 +113,13 @@ async function runManualTest(button, options, expected = {}, id = undefined) {
         `Expected response ${attribute} attribute to be ${value}`
       );
     }
+    assert_idl_attribute(response, "details");
+    assert_equals(typeof response.details, "object", "Expected an object");
+    try {
```

Comment:
This will produce a less-helpful error message. Instead, just call toJSON(), and let it throw.

Reply:
good point. Will send as a new PR.